serverless.yml

```diff
@@ -0,0 +1,54 @@
+# Welcome to serverless. Read the docs
+# https://serverless.com/framework/docs/
+
+# Serverless.yml is the configuration the CLI
+# uses to deploy your code to your provider of choice
+
+# The `service` block is the name of the service
+service: hathor-explorer-service
+
+# The `provider` block defines where your service will be deployed
+provider:
+  name: aws
+  runtime: python3.8
+  profile: serverless
+  region: us-east-1
+  vpc:
+    securityGroupIds:
+      - sg-096e4b4e
+    subnetIds:
+      - subnet-06585a4c
+      - subnet-26255a7a
+      - subnet-2ecfbc49
+
+plugins:
+  - serverless-offline
+
+package:
+  individually: true
+
+functions:
+  network_handler:
+    handler: handlers/network_handler.handle
+    package:
+      patterns:
+        - '!*'
+        - '!**/*'
+        - 'handlers/network_handler.py'
+    events:
+      - http:
+          path: network/{hash?}
+          method: get
+          cors: true
```

Comment:
Should we add just our explorer domain to the CORS?

Reply:
I don't think so. It came with first version of the file in serverless setup. Forgot to remove.